mobile/src/utils/utils.ts

```diff
@@ -0,0 +1,10 @@
+import { AppState } from '../reducers/app.reducer';
+import { Employee } from '../reducers/organization/employee.model';
+import { Optional } from 'types';
+
+//----------------------------------------------------------------------------
+export function getEmployee(state: AppState): Optional<Employee> {
```

Comment:
Utils, as well as Helper classes tend to become huge and unusable, because whenever someone doesn't now where to put a code, he creates one. And nobody looks inside to check if there is an util for it. Just buzzing.